pygmt/clib/session.py

```diff
@@ -1738,6 +1738,119 @@ def read_virtualfile(
         dtype = {"dataset": _GMT_DATASET, "grid": _GMT_GRID}[kind]
         return ctp.cast(pointer, ctp.POINTER(dtype))
 
+    def return_table(
```

Comment:
The method name `return_table` was initially proposed in https://github.com/GenericMappingTools/pygmt/issues/1318#issuecomment-861402119, but is it a good name? At line 1845, we use `kind="dataset"`, so maybe rename it to `return_dataset` or `output_dataset`? In the future, I think we will add more methods that return a grid/image/cpt/cube, so consistent method names are preferred.

Reply:
I think `return_dataset` is better than `return_table`. Renamed in ce029b252a8b29dd36dcad2bfeedef36a88c8ee8.

Reply:
How about `virtualfile_to_dataset`, or `vfile_to_dataset` (shorter)?

Reply:
`virtualfile_to_dataset`/`vfile_to_dataset` sounds a good name. I like `vfile_to_dataset`, but do we want to also rename other functions for consistency?

Reply:
You mean rename the `virtualfile_from_*` methods? Let's maybe not do that (lazy to deprecate more names). We can go with `virtualfile_to_dataset` for consistency.

Reply:
I meant to rename the `virtualfile_in`/`virtualfile_out` to `vfile_in`/`vfile_out`.

- `virtualfile_from_*` functions are no longer used in our PyGMT wrappers after #949, so it's OK to keep them unchanged.
- `virtualfile_out` was added in #3057, so we can still rename it without deprecations.
- `virtualfile_in` was renamed from `virtualfile_from_data` recently (PR #3068), so we can rename it to `vfile_in` without more deprecations.

Reply:
Let's keep with the long name (`virtualfile_to_dataset`). Renaming `virtualfile_out` -> `vfile_out` and `virtualfile_in` -> `vfile_in` will be more work and confusing (we'll need to manually update the changelog).

Reply:
Done in 711142c8816c91f7de51ebfb548990ce3a51731e.